app/views/assets/_asset.html.erb

```diff
@@ -51,10 +51,10 @@
 
 <div class="tracks_reveal" data-target="normal-playback.details">
   <% if logged_in? %>
-    <%= link_to toggle_favorite_path(asset_id: asset.id), remote: true, class: 'add_to_favorites',
-      data: {controller: 'favorite', action: 'favorite#toggle' } do %>
-      <%== render file: svg_path('svg/animation_heart.svg') %>
-    <% end %>
+    <a href="#" data-remote="true", class='add_to_favorites'
```

Comment:
Just a friendly heads up that @ofsound has a branch open that includes layout/css changes to the favorite button (so you aren't surprised if yours ends up merging afterwards) https://github.com/sudara/alonetone/pull/940/files#diff-1ab2e2c4523e138fdf8a768d7f67b9caL21